capsules/src/virtual_i2c.rs

```diff
@@ -193,7 +193,7 @@ impl<'a, I: i2c::I2CMaster> MuxI2C<'a, I> {
     }
 }
 
-impl<'a, I: i2c::I2CMaster> DynamicDeferredCallClient for MuxI2C<'a, I> {
+impl<'a, I: i2c::I2CMaster, S: i2c::SMBusMaster> DynamicDeferredCallClient for MuxI2C<'a, I, S> {
```

Comment:
```suggestion
impl<'a, I: i2c::I2CMaster, S: i2c::SMBusMaster = NoSMBus> DynamicDeferredCallClient for MuxI2C<'a, I, S> {
```

Reply:
This gives me the following error: `error: defaults for type parameters are only allowed in 'struct', 'enum', 'type', or 'trait' definitions`